applications/EmpireApplication/tests/sdof_static_solver/naca0012_small_parameters_coupling.json

```diff
@@ -0,0 +1,187 @@
+{
+    "problem_data"                     : {
```

Comment:
As far as I see in this test you are testing not only the static sdof solver but the whole FSI workflow with the potential solver. Therefor I would suggest renaming this test to: fsi_potential_flow_sdof and adding another test where you only test the static sdof solver as it is done as it is done in the sdof_solver (i.e. check sdof_solver)

Reply:
done